src/features/surveys/components/SurveyEditor/blocks/OpenQuestionBlock.tsx

```diff
@@ -1,13 +1,235 @@
+  MenuItem,
+  TextField,
+  Typography,
+} from '@mui/material';
+
+type TextQuestionPatchBody = Omit<ZetkinTextQuestion, 'required'>;
 
 interface OpenQuestionBlockProps {
-  question: ZetkinTextQuestion;
+  element: ZetkinTextQuestion;
+  inEditMode: boolean;
+  onEditModeEnter: () => void;
+  onEditModeExit: (question: TextQuestionPatchBody) => void;
+}
+
+enum FIELDTYPE {
+  MULTILINE = 'multiLine',
+  SINGLELINE = 'singleLine',
 }
 
-const OpenQuestionBlock: FC<OpenQuestionBlockProps> = ({ question }) => {
-  return <Box>{JSON.stringify(question)}</Box>;
+const OpenQuestionBlock: FC<OpenQuestionBlockProps> = ({
+  element,
+  inEditMode,
+  onEditModeEnter,
+  onEditModeExit,
+}) => {
+  const intl = useIntl();
+
+  const [typeField, setTypeField] = useState(
+    element.response_config.multiline === true
+      ? FIELDTYPE.MULTILINE
+      : FIELDTYPE.SINGLELINE
+  );
+
+  const [title, setTitle] = useState(element.question);
+  const [description, setDescription] = useState(element.description);
+
+  const [responseConfig, setResponseConfig] = useState(
+    element.response_config.multiline
+  );
+  const [focus, setFocus] = useState<
+    'title' | 'description' | 'responseConfig' | null
+  >(null);
+
+  const titleRef = useCallback((node: HTMLInputElement) => {
+    node?.focus();
+  }, []);
+  const descriptionRef = useRef<HTMLInputElement>(null);
+  const typeConfigRef = useRef<HTMLInputElement>(null);
+
+  useEffect(() => {
+    if (focus === 'description') {
+      const input = descriptionRef.current;
+      input?.focus();
+    } else if (focus === 'responseConfig') {
+      const input = typeConfigRef.current;
+      input?.focus();
+    }
+  }, [focus]);
+
+  const handleKeyDown = (evt: KeyboardEvent<HTMLDivElement>) => {
+    if (evt.key === 'Enter') {
+      const element = createElementToUpdate();
+      onEditModeExit(element);
+      setFocus(null);
+    }
+  };
+
+  const handleSelect = (event: BaseSyntheticEvent) => {
+    setTypeField(event.target.value);
+  };
+
+  const createElementToUpdate = () => {
+    const elemenToUpdate: TextQuestionPatchBody = {
+      description: description,
+      question: title,
+      response_config: { multiline: responseConfig },
+      response_type: element.response_type,
+    };
+    return elemenToUpdate;
+  };
+
+  return (
+    <>
+      {inEditMode && (
+        <ClickAwayListener
+          onClickAway={() => {
+            const element = createElementToUpdate();
+            onEditModeExit(element);
+            setFocus(null);
+          }}
+        >
+          <Box display="flex" flexDirection="column">
+            <TextField
+              defaultValue={element.question}
+              fullWidth
+              InputProps={{
+                inputRef: titleRef,
+                sx: { fontSize: theme.typography.h4.fontSize },
+              }}
+              label={intl.formatMessage({
+                id: 'misc.surveys.blocks.common.empty',
+              })}
```

Comment:
I think the label needs to be more specific, in the design it's "Title". I'm guessing this is a copy paste mistake?